flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/StreamingFileSink.java

```diff
@@ -128,24 +128,12 @@
 	private transient ListState<Long> maxPartCountersState;
 
 	/**
-	 * Creates a new {@code StreamingFileSink} that writes files in row-based format to the given base directory.
+	 * Creates a new {@code StreamingFileSink} that writes files to the given base directory
```

Comment:
I think it should not affect compatibility since it is only called from `BucketsBuilder`. Besides, it should be only extends its ability to accept more types of `BucketsBuilder`.